stackrox/base/policy/networkpolicy.yaml

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Let's split these networkPolicies up too. It just makes it easier to see the that we have multiple objects

Reply:
will come back to this in an hour

Reply:
in progress -> separating network resources

Reply:
completed